spec/features/user_visits_root_spec.rb

```diff
@@ -11,8 +11,9 @@
   # * Visitor must see a list of categories that are links to other pages
 
   scenario 'user visits root path' do
+    user = FactoryGirl.create(:user)
     category = FactoryGirl.create(:category)
-    item = FactoryGirl.create(:item, category: category)
+    item = FactoryGirl.create(:item, user: user, category: category)
 
     visit root_path
     expect(page).to have_content("Sign In")
```

Comment:
It looks like you might have two spec files that are testing the same thing. 

`user_views_items_index_spec.rb` and
`user_visits_root_path_spec.rb`

I would change the scenario names on these so it is more clear as to what is being tested.